app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -42,19 +38,23 @@
   }
 }
 
-span.preview_text_element_name {
+.preview_text_element_name {
```

Comment:
Selector `preview_text_element_name` should be written in lowercase with hyphens